rules/integrations/aws/ml_cloudtrail_rare_error_code.toml

```diff
@@ -23,19 +23,81 @@ interval = "15m"
 license = "Elastic License v2"
 machine_learning_job_id = "rare_error_code"
 name = "Rare AWS Error Code"
-note = """
-## Config
+note = """## Triage and analysis
 
-The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
+### Investigating Rare AWS Error Code
+
+CloudTrail logging provides visibility on actions taken within an AWS environment. By monitoring these events and
+understanding what is considered normal behavior within an organization, you can spot suspicious or malicious activity
+when deviations occur.
+
+This rule uses a machine learning job detected an unusual error in a CloudTrail message. These can be byproducts of
+attempted or successful persistence, privilege escalation, defense evasion, discovery, lateral movement, or collection.
+
+Detection alerts from this rule indicate a rare and unusual error code that was associated with the response to an AWS
+API command or method call.
+
+#### Possible investigation steps
+
+- Examine the history of the error, and if the error, which is visible in the `aws.cloudtrail.error_code field`, only
+manifested recently, it might be related to recent changes in an automation module or script.
```

Comment:
```suggestion
manifested recently, it might be related to recent changes in an automation module or script. You can find the error in the `aws.cloudtrail.error_code field` field.
```